client/web/src/site-admin/SiteAdminWebhooksPage.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useEffect } from 'react'
+
+import { mdiMapSearch, mdiPlus } from '@mdi/js'
+import { RouteComponentProps } from 'react-router'
+import { WebhookFields } from 'src/graphql-operations'
+
+import { TelemetryProps } from '@sourcegraph/shared/src/telemetry/telemetryService'
+import { ButtonLink, Container, H5, Icon, PageHeader } from '@sourcegraph/wildcard'
+
+import { FilteredConnection } from '../components/FilteredConnection'
+import { PageTitle } from '../components/PageTitle'
+
+import { queryWebhooks } from './backend'
+import { WebhookNode } from './WebhookNode'
+
+import styles from './SiteAdminWebhooksPage.module.scss'
+
+interface Props extends RouteComponentProps<{}>, TelemetryProps {}
+
+export const SiteAdminWebhooksPage: React.FunctionComponent<React.PropsWithChildren<Props>> = ({
+    history,
+    location,
+    telemetryService,
+}) => {
+    useEffect(() => {
+        telemetryService.logPageView('SiteAdminWebhooks')
+    }, [telemetryService])
+
+    const queryConnection = useCallback(() => queryWebhooks(), [])
+
+    return (
+        <div className="site-admin-webhooks-page">
+            <PageTitle title="Webhook receivers" />
+            <PageHeader
+                path={[{ text: 'Webhook receivers' }]}
+                headingElement="h2"
+                description="All configured webhooks receivers"
+                className="mb-3"
+                actions={
+                    <>
+                        <ButtonLink className="test-create-webhook" variant="primary">
+                            <Icon aria-hidden={true} svgPath={mdiPlus} /> Add webhook
+                        </ButtonLink>
+                    </>
```

Comment:
You have a single element here, you don't need to wrap it in a fragment (which you'd need if you want to have multiple elements here):

```suggestion
                        <ButtonLink className="test-create-webhook" variant="primary">
                            <Icon aria-hidden={true} svgPath={mdiPlus} /> Add webhook
                        </ButtonLink>
```

Reply:
Thanks, will fix that!